File: frontend/src/app/domain/Product.ts
export class Product {
  constructor(public productId: string,
              public title: string,
              public imgUrl: string,
              public imgFileId: string,
              public price: number,
              public introduction: string,
              public detail: string,
              public bought: boolean,
              public soldNumber: number) {
  }
}

File: src/main/java/com/netease/yinanmall/pojo/Product.java
package com.netease.yinanmall.pojo;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

/**
 * @author yinan
 */
@Document
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Product {
    @Id
    private String productId;

    private String title;

    private String imgUrl;

    private Double price;

    private String introduction;

    private String detail;

    private Boolean bought;

    private Integer soldNumber;
}
